docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -928,48 +928,75 @@ Additionally, we would like the *number* of simulants assigned to each treatment
+incorporate 4, personal history of malignancy, as a determinant in registry 
+enrollment within the simulation as it introduced additional analytic complexity 
+and we expect this criterion to have a small impact on eligibility.  
 
 .. note::
  
- The eligibility of Isatuximab treatment might change based on the guidance from our clients.
+ We chose not to enforce inclusion criterion number 2 (with RRMM) for the 
+ alternative scenario in our simulation. The removal of this criterion allows us 
+ to demonstrate the potential additional number of eligible and enrolled patients 
+ if the registry were to accept NDMM patients at their first line of treatment in 
+ accordance with the modeled isatuximab scale-up in the first line of treatment 
+ in our alternative scenario.
 
 Power calculation
 ^^^^^^^^^^^^^^^^^
 
-To calculate the number of simulants in the registry for each calendar year from 
-2021 to 2025, we will use the equation presented below: 
+To calculate the number of simulants eligible for inclusion and the number of 
+simulants in the registry for each calendar year from 2021 to 2025, we used the 
+equation presented below: 
+
+:math:`N_{candidate}(t) = Prev_{RRMM}(t) \times F_{Isa}`
+
+And,
 
-:math:`N_{enroll}(t) = N_{0} + Prev_{RRMM}(t) \times F_{Isa} \times P`
+:math:`N_{enrollee}(t) = N_{candidate}(t) \times P`
 
 Where,
- - :math:`N_{enroll}(t)` is the number of simulants in the registry in year t.
- - :math:`N_{0}` is the number of simulants in the registry at the beginning of 
-   the simulation. (2021-01-01)
+ - :math:`N_{candidate}(t)` is the number of registry candidates in year t.
+ - :math:`N_{enrollee}(t)` is the number of simulants in the registry in year t.
  - :math:`Prev_{RRMM}(t)` is the number of adult (15+ in our sim) prevalent RRMM 
    cases in year t.
  - :math:`F_{Isa}` is the proportion of population eligible for Isatuximab treatment.
  - :math:`P` is the proportion of registry candidates that will be selected into 
    the registry. We assume P is a constant number equal to **5%** over time.
 
+Registry enrollment system
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We set January 1, 2021 as the start date of the registry and assumed that no 
+one was in the registry prior to the start date. At each simulation time step, 
+we looked at each simulant’s disease status and treatment history. If an 
+individual simulant met the inclusion criteria, they were eligible for inclusion. 
+Those eligible had a 5% chance of being enrolled, and they would stay in the 
+registry until death. The probability used to enroll eligible simulants into the 
+registry is a constant variable over the course of the simulation, independent 
+from simulant’s age, sex, race, cytogenetic risk, and renal function. We captured 
+two measures from the patient registry component: the cumulative number of eligible 
+simulants; and the cumulative number of living simulants in the registry.  
```

Comment:
I read this to mean that a simulant will stay in the registry from enrollment until death (or end of simulation), but it occurs to me that @mattkappel might have interpreted it to mean the simulant leaves the registry if/when they progress to a new line of non-isa treatment.  @yongqx : which did you intend?  and @mattkappel : which did you implement?

Reply:
@aflaxman I think we used `registry_status_newly_enrolled`, a measure of count to track how many simulants were enrolled each year; and `registry_status_enrolled`, a measure of person-time to track the length of time those simulants spent in the registry. In regards to person-time, i think it is upon death, rather than new line of treatment. @mattkappel correct me if i was wrong on this.

Reply:
You are correct, once added to the registry, only death removes.

Reply:
Thanks for confirming it!